winter-website.css

```diff
@@ -0,0 +1,171 @@
+body {
+	padding: 0px;
+	margin: 0px;
+	color: #e8f3fc;
+	background-color: black;
+	font-family: sans-serif;
+}
+
+*, *::before, *::after {
+   box-sizing: border-box;
+}
+
+.navBar {
+	display: flex;
+	justify-content: space-between;
+	width: auto;
+	padding: 20px;
+	margin: 20px;
+	background-color: #5d87cf;
+	opacity: 0.5;
+}
+
+a.navLink {
+	display: flex;
+	color: #e8f3fc;
+	text-decoration: none;
+}
+
+.navLink:hover {
+	color: black;
+}
+
+h1 {
+	font-size: 8rem;
+	padding: 0;
+	margin: 0;
+}
+
+.sectionAll {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	position: relative;
+	flex-wrap: wrap;
+	width: auto;
+	padding: 20px;
+	margin: 20px;
+}
+
+.aboutMeSection {
+	min-width: 200px;
+	position: relative;
+	width: 50%;
+	padding: 20px;
+	margin: 20px;
+}
+
+h2 {
+	padding: 0;
+	margin: 0;
+}
+
+.aboutMeImages {
+	display: flex;
+	flex-wrap: wrap;
+	min-width: 500px;
+	max-width: 600px;
+	padding: 20px;
+	margin: 20px;
+}
+
+.aboutMeImages img {
+	width: 40%;
+	height: auto;
+	padding: 10px;
+}
+
+.footerBar {
+	position: relative;
+	width: auto;
+	color: #e8f3fc;
+	opacity: 0.75;
+}
+
+@media screen and (max-width: 760px) {
+	
+	.navBar {
+		display: flex;
+		flex-direction: row;
+		opacity: 0.75;
+	}
+	
+	h1 {
+		font-size: 5rem;
+		padding: 0;
+		margin: 0;
+	}
+	
+	.sectionAll {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		position: relative;
+		flex-wrap: wrap;
+		width: auto;
+		padding: 20px;
+		margin: 20px;
+	}
+}
+
+@media screen and (max-width: 590px) {
+	
+	body {
+		
+	}
+	
+	.navBar {
+		display: flex;
+		flex-direction: column;
+		min-width: 30%;
+		width: 30%;
+		opacity: 1;
+		font-size: 80%;
```

Comment:
i've not seen this before, i am sure its fine to use though i think its better to be consistent.
use rem??

Reply:
rem used for font